config/config.go

```diff
@@ -21,6 +21,9 @@ type NodeConfig interface {
 
 	// block storage
 	BlockSyncCommitTimeoutMillis() time.Duration
+	BlockTransactionReceiptQueryStartGraceSec() time.Duration
```

Comment:
It's not consistent that some of the time related configs we have are in `uint32` (like `BenchmarkConsensusRoundRetryIntervalMillis`) and some are in `time.Duration` (like `BlockTransactionReceiptQueryStartGraceSec`).

We should choose one convention

Personally I prefer `uint32` because when we have config providers such as json files or smart contracts, primitive types are easier to work with and go-specific constructs like `time.Duration` are language-specific. In the jsons we're going to have time in `uint32` and I usually like these configs to be closer to the code

Reply:
i agree on all comments, but since we *must* refactor the configuration, and next week (end of week) we will have a new version, how important (1 to 10) that i refactor this here?